tasks/install.yml

```diff
@@ -1,12 +1,9 @@
 ---
 
- - name: Disable default postgres packages
-  copy:
-    src: exclude-default-postgres.repo
-    dest: /etc/yum.repos.d/exclude-default-postgres.repo
-    owner: root
-    group: root
-    mode: 0644
+- name: Disable default postgres packages
```

Comment:
Not a huge deal, but I might change this to something like, "Verify that we're not using default PostgreSQL packages."

Reply:
I agree. I went ahead and changed that.